openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -5992,14 +5992,31 @@ OS:OtherEquipment,
        \reference SpaceItemNames
        \reference SpaceLoadNames
        \reference SpaceComponentInstanceNames
-  A3, \field Other Equipment Definition Name
+  A3,  \field Other Equipment Definition Name
        \type object-list
        \required-field
        \object-list OtherEquipmentDefinitionNames
-  A4, \field Space or SpaceType Name
+  A4,  \field Fuel Type
```

Comment:
@nmerket You need to move this to the end of the existing object, otherwise you will need to write a version translation.